examples/clone/clone.bal

```diff
@@ -0,0 +1,69 @@
+import ballerina/io;
+
+//Define anydata record
+public type Person record {
+    string name;
+    int age;
+    boolean married;
+    float salary;
+    Address address;
+};
+
+public type Address record {
+    string country;
+    string state;
+    string city;
+    string street;
+};
+
+// The returned tuple will be later used by ballerina tests
+public function main() returns (Person, Person, string)? {
+    // Create a value of Address
+    Address address = {
+        country : "USA",
+        state: "NC",
+        city: "Raleigh",
+        street: "Daniels St"
+    };
+
+    // Create a value of type Person
+    Person person = {
+        name: "Alex",
+        age: 24,
+        married: false,
+        salary: 8000.0,
+        address: address
+    };
+
+    // Note that we have assigned the person value to any typed variable. So the type of the value has to be determined
```

Comment:
Shall we shorten this and just say "Assign the Person value..."

Reply:
I changed it to "// Assign the person value to anyValue. So the type of the value has to be determined in runtime.
    ". Can you please review. I think we have to specifically say why we need to assign the Person to anyValue

Reply:
Yes, this is fine.